### PR TITLE
feat: Access dcc and update masa package and update table structure

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/Services/MASA.PM.Service.Admin/Program.cs [hunk 1]
```diff
@@ -33,13 +28,17 @@
...
+builder.AddMasaConfiguration(configurationBuilder => configurationBuilder.UseDcc());
+
 builder.Services.AddMasaIdentityModel(IdentityType.MultiEnvironment, options =>
 {
     options.Environment = "environment";
     options.UserName = "name";
     options.UserId = "sub";
 });
-builder.Services.AddAuthClient(builder.Configuration["AuthServiceBaseAddress"]);
+builder.Services.AddAuthClient(builder.GetMasaConfiguration().ConfigurationApi.GetDefault()["Appsettings:AuthServiceBaseAddress"]);
```
[the remark this answers is not in the document]
Comment (codding-y) [reply src/Services/MASA.PM.Service.Admin/Program.cs:39]: Defalut=>AppId
Public=>PublicId

[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/MASA.PM.Service.Admin/Program.cs:39]: Default -> CurrentApp, and change it to property.

File: src/Services/MASA.PM.Service.Admin/Program.cs [hunk 2]
```diff
@@ -1,10 +1,7 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
 
-using Masa.BuildingBlocks.Identity.IdentityModel;
-using Masa.Contrib.Data.Contracts.EF;
-using Masa.Contrib.Data.EntityFrameworkCore.SqlServer;
-using MASA.PM.Service.Admin.Migrations;
+using Dapr.Client;
```
Comment (doddgu) [inline src/Services/MASA.PM.Service.Admin/Program.cs:4]: Move it to _Imports.

File: src/Services/MASA.PM.Service.Admin/Program.cs [hunk 3]
```diff
@@ -33,13 +30,39 @@
 }
 #endregion
 
+if (builder.Environment.IsProduction())
+{
+    //var daprClient = GetDaprClient(builder.Services);
```
Comment (doddgu) [inline src/Services/MASA.PM.Service.Admin/Program.cs:35]: If you don't need it anymore, you can remove it.  if you just comment it to test, you should add comment tell me.

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**